machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -47,15 +47,39 @@ It is computed using the solutions from the L0 and L1 grids with the equation
 .. math::
     f_{h=0} = f_{L0} + \frac{f_{L0}-f_{L1}}{r^{\hat{p}} - 1}
 
-Grid refinement study with ADflow
----------------------------------
+Grid Refinement Study on Airfoils
+======================================
```

Comment:
I would revert to `--------------` because this section should be a level below the header of this entire page. Right now this is also messing with the left-side index levels